bifunctors.cabal

```diff
@@ -54,42 +39,27 @@ flag tagged
 
 library
   hs-source-dirs: src
-  include-dirs: include
-  includes: bifunctors-common.h
   build-depends:
-    base                >= 4.3   && < 5,
+    base                >= 4.9   && < 5,
     base-orphans        >= 0.8.4 && < 1,
     comonad             >= 5.0.7 && < 6,
     containers          >= 0.2   && < 0.7,
```

Comment:
This lower bound could be 0.5.7.1 (which is bundled with GHC-8.0.2). There doesn't seem to be install plans with older containers anyway.

Reply:
I've bumped the lower bound on `containers` to `0.5.7.1`.